.taskcat.yml

```diff
@@ -30,6 +30,7 @@ tests:
       TAPClusterArch: single
     regions:
       - us-east-1
+      - eu-south-1
```

Comment:
I advise against "blindly" adding regions, without further considerations.

As our pipeline stands right now, it assumes each tastkat test to run in one region. This is a deliberate random choice I made when building the pipeline. The reasoning is, that I think it is useful to be able to (re) run a single test in a single region by just hitting <img width="32px" src="https://user-images.githubusercontent.com/1401333/221535242-5bcb5c25-3c8d-4eb0-904f-a5285c552788.png" /> rerun on the pipeline job ; thus parallelising test runs in the pipeline and not in taskcat.  

Another thing we should ensure when expanding to other regions is that the cleanup / housekeeping jobs run there. Otherwise we'd be somewhat blind about failing resources there.

Also, as we
- run `taskcat upload` in separation
- we upload (cached) lambda zips out of band
- and we use [regional buckets](https://github.com/vmware-tap-on-public-cloud/quickstart-vmware-tanzu-application-platform/blob/main/.taskcat.yml#LL4C7-L4C7)

we need to check how taskcat handles all this. With limited understanding how taskcat works internally, that's something I can't answer from the top of my head.

And then we'd need to [ensure quotas are set correctly](https://confluence.eng.vmware.com/pages/viewpage.action?pageId=1574466417#:~:text=%E2%9C%85-,Bump%20quotas,-%3A) to successfully deploy (multiple stacks) in that "new" region.

So I suggest to:
- remove the additional region from here / in this PR
- make a decision if we want taskcat to parallelize tests across regions or if we want concourse to do that
- adapt the pipeline, potentially the taskcat_overlay.yml, potentially the tasks, so that they can handle whatever decision we made

Reply:
Thank you for these considerations. I agree that for this PR i should remove the new test region and we can create independent stories related to multi region tests in CI

Reply:
The upstream repo uses this .taskcat.yml for CI. Here our local CI differs from upstream.
I prefer to have this test in the upstream repo as well.
Please remove it if it is hard to handle this in our local CI.

Reply:
> Please remove it if it is hard to handle this in our local CI.

It's not hard to handle. But it needs to be handled.

@leclaist let me know if you want to do this in this PR or in a follow-up and if you need help.